test/bulk-info.test.js

```diff
@@ -102,7 +105,31 @@ describe('Test bulk info plugin', () => {
           return Number.isNaN(num) ? false : num === 2;
         }),
       }).run();
+
       assert.ok(sizeCheck, 'Wrong selection size displayed');
     }).timeout(IT_DEFAULT_TIMEOUT);
+
+    // Test case for non-Latin SharePoint item info
+    it(`Bulk info plugin handles non-Latin SharePoint item info in ${env} list view`, async () => {
+      nock.sidekick();
+      nock.admin(setup, {
+        route: 'status',
+        type: 'admin',
+      });
+      const { checkPageResult: textContent } = await new SidekickTest({
+        browser,
+        page,
+        fixture: languageFixture,
+        url: setup.getUrl('edit', 'admin'),
+        loadModule: true,
+        checkPage: (p) => p.evaluate(() => {
+          const infoElement = document.getElementById('hlx-sk-bulk-info');
+          return infoElement.textContent;
+        }),
+      }).run();
+
+      const expectedTextContent = 'heroes.docx, docx 文件, 专用, 已于 4/6/2024 修改, 编辑者: Firstname, Lastname, 875 KB';
```

Comment:
You are testing if the static content matches this string, but not if the sidekick correctly picked up the filename and type for the selection.

Reply:
as discussed, this is updated